minebooks2.py

```diff
@@ -0,0 +1,205 @@
+
+def RunSentAnalysis(mylist):
+	"""
+	runs sentimient analysis and returns positive and negitive sentiments in a list
+	"""
+	res = []
+	analyzer = SentimentIntensityAnalyzer()
+	answer = analyzer.polarity_scores(mylist)
+	res.append(answer["pos"])
+	res.append(answer["neg"])
+	return res
+	
+
+def Most_Common(text):
+	"""
+	takes a string and returns the 25 most common words in the string and returns a list of those words
+	"""
+	top_words = []
+	freqwords = defaultdict(int) #initializes a dictionary
+	
+	for words in text.split(): #goes through a list of all the words in the string text
+		freqwords[words] += 1  #if a word is found add 1 to the counter
+	sortedwords = sorted(freqwords.items(), key=operator.itemgetter(1), reverse = True) #sorts dictionary of words based on how many times the word was found in reverse order. 
+	for i in range(25):
+		top_words.append(sortedwords[i][0])	#store the 25 most common words
+	return (top_words)						#Rreturns a list of the 24 most common words
+
+def commonOverAll(list1,list2,list3):
+	""" 
+	returns a list of all the most universally common elimments of three lists. In the program this is used to find all of universaly
+	common words in the three play types.
+
+	"""
+	
+	return( list(set(list(set(list1).intersection(list2))).intersection(list3)))
+
+def removewords(words,wordstoremove): 
+	"""
+	Removes words from the plays that are univeraly common amonge all types
+	"""
+	words = words.split()
+	return ([x for x in words if x not in wordstoremove]) #return elements in words that are not in wordstoremove
+
+def listtostring(mylist):
+	"""
+	convers a list of words to string of words
+	"""
+	return ' '.join(mylist)
+
+def linklists(mylist):
+	"""
+	Add several list together into one large list
+	"""
+	res = []
+	for i in range(len(mylist)):
+		res += [mylist[i]]
+	return listtostring(res)
+
+def sampling(mylist, trials):
+	"""
+	Takes a random sample of ten words from my list and preforms 
+	sentiment analysis on them. Averages all the positive and 
+	the negative across a many trials. Returns the list with
+	the average positive and negative intensities. 
+	"""
+	pos = 0
+	neg = 0
+	for i in range(trials):
+		sample = listtostring(random.sample(mylist,10))
+		sentiment = RunSentAnalysis(sample)
+		pos += sentiment[0]
+		neg += sentiment[1]
+
+	return[pos/trials, neg/trials]
+
+
+def textmining(): # Main function that runs the textmining code.
+	"""
+	13 of Shakespeare’s plays were saved from gutenberg.org.
+	I sorted the file names of all of the plays into tree lists
+	comedies, tragedies, and histories.
+
+	"""
+	
+	comedies = ['A_Midsummer_Nights_Dream.pickle', 'Alls_Well_That_Ends_Well.pickle']
+	tragedies = ['Antony_and_Cleopatra.pickle']
+	tragedies.append('Coriolanus.pickle')
+	tragedies.append('Cymbeline.pickle')
+	histories = ['King_Henry_IV.pickle']
+	histories.append('King_John.pickle')
+	histories.append('King_Richard_II.pickle')

```

Comment:
I'm a little confused about why you appended these strings instead of just writing them out initially.
Do this:
`tragedies = ['Antony_and_Cleopatra.pickle', 'Coriolanus.pickle', 'Cymbeline.pickle']`
instead of:
`tragedies = ['Antony_and_Cleopatra.pickle']
tragedies.append('Coriolanus.pickle')
tragedies.append('Cymbeline.pickle')`